src/annotator/config/index.js

```diff
@@ -51,3 +96,23 @@ export default function configFrom(window_) {
     ),
   };
 }
+
+/**
+ * Return the configuration for a given application context.
+ *
+ * @param {'sidebar'|'notebook'|'annotator'|'all'} [appContext] - The name of the app.
+ */
+export default function getConfig(appContext = 'annotator', window_ = window) {
+  // Filter the config based on the application context as some config values
+  // may be inappropriate or erroneous for some applications.
+  let partialConfig = {};
+  const fullConfig = configFrom(window_);
```

Comment:
`configFrom` presumably ends up doing some redundant work fetching values for keys which are never used. Perhaps it could accept a whitelist of keys as an argument?

Reply:
Sure, but in the next PR or two, I will propose this function goes away. So I hope not to get too attached to details with configFrom(). Part of what has been hard was landing intermittent changes that make sense.